3.interpret_models/linear_reg_plate_1_2_norm_data/feature_importances/nbconverted/seperate_compartments.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Here the features are seperated according to compartments for later analysis
+
+# In[1]:
+
+
+# %%--%%| <qGnYViiwRD|SyZ3qa8iz3>
+r"""°°°
+## Imports
+°°°"""
+
+
+# In[ ]:
+
+
+import pandas as pd
+import numpy as np
+from pathlib import Path
+from sklearn.preprocessing import LabelEncoder
+from sklearn.linear_model import LogisticRegression
+
+from sklearn.metrics import (
+    confusion_matrix,
+    ConfusionMatrixDisplay,
+    precision_score,
+    accuracy_score,
+)
+from sklearn.model_selection import train_test_split
+
+import matplotlib.pyplot as plt
+import seaborn as sns
+import itertools
+
+from joblib import dump, load
```

Comment:
same comment about `pycln` pre-commit here

Reply:
I ended up using a temporary workaround for this, I'm going to try vulture.